api/v1alpha1/spacerequest_types.go

```diff
@@ -4,6 +4,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// SpaceRequestLabelKey is a label on the subSpace, and will hold the name of the SpaceRequest that created the subSpace resource.
+	SpaceRequestLabelKey = LabelKeyPrefix + "spaceRequest"
+	// SpaceRequestNamespaceLabelKey is a label on the subSpace, and will hold the namespace of the SpaceRequest that created the subSpace resource.
+	SpaceRequestNamespaceLabelKey = LabelKeyPrefix + "spaceRequestNamespace"
```

Comment:
I don't think that using the camel-case is a good approach for label keys (and probably also values)
```suggestion
	// SpaceRequestLabelKey is a label on the subSpace, and will hold the name of the SpaceRequest that created the subSpace resource.
	SpaceRequestLabelKey = LabelKeyPrefix + "spacerequest"
	// SpaceRequestNamespaceLabelKey is a label on the subSpace, and will hold the namespace of the SpaceRequest that created the subSpace resource.
	SpaceRequestNamespaceLabelKey = LabelKeyPrefix + "spacerequest-namespace"
```

Reply:
I saw now that was not following the standards ! 

Applied your suggestion! Thanks!